apps/studio/components/interfaces/Auth/Hooks/CreateHookSheet.tsx

```diff
@@ -0,0 +1,352 @@
+              name="selectedType"
+              render={({ field }) => (
+                <FormItemLayout label="Hook type" className="px-8">
+                  <FormControl_Shadcn_>
+                    <Radio.Group
+                      type="cards"
+                      value={field.value}
+                      onChange={(e) => field.onChange(e.target.value)}
+                    >
+                      <Radio
+                        key="postgres"
+                        id="postgres"
+                        value="postgres"
+                        label=""
+                        beforeLabel={
+                          <div className="flex items-center space-x-5">
+                            <div className="flex-col space-y-0">
+                              <div className="flex space-x-2">
+                                <p className="text-foreground">Postgres</p>
+                              </div>
+                              <p className="text-foreground-light">
+                                Used to call a Postgres function.
+                              </p>
+                            </div>
+                          </div>
+                        }
+                      />
+                      <Radio
+                        key="https"
+                        id="https"
+                        value="https"
+                        label=""
+                        beforeLabel={
+                          <div className="flex items-center space-x-5">
+                            <div className="flex-col space-y-0">
+                              <div className="flex space-x-2">
+                                <p className="text-foreground">HTTPS</p>
+                              </div>
+                              <p className="text-foreground-light">
+                                Used to call any HTTPS endpoint.
+                              </p>
+                            </div>
+                          </div>
+                        }
+                      />
+                    </Radio.Group>
+                  </FormControl_Shadcn_>
+                </FormItemLayout>
+              )}
+            />
+            {values.selectedType === 'postgres' ? (
+              <>
+                <FormField_Shadcn_
+                  key="postgresValues.schema"
+                  control={form.control}
+                  name="postgresValues.schema"
+                  render={({ field }) => (
+                    <FormItemLayout
+                      label="Schema"
+                      className="px-8"
+                      description="Postgres schema where the function is defined."
+                    >
+                      <FormControl_Shadcn_>
+                        <SchemaSelector
+                          size="small"
+                          showError={false}
+                          selectedSchemaName={field.value}
+                          disabled={field.disabled}
```

Comment:
you don't need to do this

just `{...field}`

Reply:
Fixed.